assignment6/rearranging_cars.py

```diff
@@ -0,0 +1,71 @@
+
+class Movement(object):
+    """This class stores information about one movement,
+    i.e. a car moving from position *fro* to position *to*.
+    """
+
+    def __init__(self, car, fro, to):
+        self.car = car
+        self.fro = fro
+        self.to = to
+
+    def print_movement(self):
+        print("Move car %s from %s to %s" %(self.car, self.fro, self.to))
+
+def rearrange(current_permutation, final_permutation):
```

Comment:
I guess it is worth to make a class in addition to Movement (handy class), that stores all variables like current_permutation,  final_permutation, car_to_pos to avoid passing them to the functions:)